test/react/rails/realtime_update_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class RealtimeUpdateTest < ActiveSupport::TestCase
+  ShakapackerHelpers.when_shakapacker_available do
+    include Capybara::DSL
+
+    def assert_counter_count(page, timer_name, count)
+      assert page.has_content?("#{timer_name} - #{count}"), <<~MSG
+        #{page.body}
+        #{page.driver.browser.logs.get(:browser).inspect}
```

Comment:
This is what's causing the log error:

```suggestion
        #{page.driver.browser.manage.logs.get(:browser).inspect}
```

Reply:
I did recommend applying it back when we were doing the CI fixing PRs, but we ended up going the other direction.

Overall that change is out of scope for this PR, but I also think it might also not be possible on the current gem versions which is why I've not done a PR myself - I'm happy to review a PR if you want to give it a try, but it'll eventually get done after dropping support for Ruby 2.7 etc if it's not done before then.